answers/Artur/stackoverflow/google/python/stackoverflow-webscraping-google-search-results-using-google-api-serpapi.py

```diff
@@ -0,0 +1,20 @@
+# https://stackoverflow.com/questions/61854976/webscraping-google-search-results-using-google-api-returns-same-result-over-an
+# https://replit.com/@chukhraiartur/stackoverflow-webscraping-google-search-results-using-google#main.py
+
+from serpapi import GoogleSearch
+import os
+
+query = input("Enter the query:\n")
+
+# You must enter your API_KEY
+params = {
+  "api_key": os.getenv("API_KEY"),
+  "engine": "google",
+  "q": query
+}
```

Comment:
```suggestion
params = {
  # https://docs.python.org/3/library/os.html#os.getenv
  "api_key": os.getenv("API_KEY"),  # your serpapi api key
  "engine": "google",               # search engine
  "q": query                        # search query
  # other parameters
}
```

Here you're describing what each parameter does and also mentioning that there are more available parameters `# other parameters`.